src/d3d9/d3d9_device.cpp

```diff
@@ -522,7 +522,8 @@ namespace dxvk {
     if (unlikely(m_losableResourceCounter.load() != 0 && !IsExtended() && m_d3d9Options.countLosableResources)) {
       Logger::warn(str::format("Device reset failed because device still has alive losable resources: Device not reset. Remaining resources: ", m_losableResourceCounter.load()));
       m_deviceLostState = D3D9DeviceLostState::NotReset;
-      return D3DERR_DEVICELOST;
+      // D3D8 expects a D3DERR_DEVICELOST error code
+      return m_isD3D8Compatible ? D3DERR_DEVICELOST : D3DERR_INVALIDCALL;
```

Comment:
... and D3D9 expects INVALIDCALL?

Reply:
Reworded it so it's a bit more clear what happens here.